utilities/test_suite/HIP/runMiscTests.py

```diff
@@ -206,8 +206,8 @@ def rpp_test_suite_parser_and_validator():
 os.chdir(buildFolderPath + "/build")
 
 # Run cmake and make commands
-subprocess.run(["cmake", scriptPath], cwd=".")   # nosec
-subprocess.run(["make", "-j16"], cwd=".")    # nosec
+subprocess.call(["cmake", scriptPath], cwd=".")   # nosec
+subprocess.call(["make", "-j16"], cwd=".")    # nosec
```

Comment:
What's the difference in changing from run to call? vs the Popen above?

Reply:
.call returns either 0 or 1 as output, in the above case we need to print the log in terminal and as well as to the log file , we need stdout to be returned from the subprocess, Popen return that log